HTML5-frontend/script.js

```diff
@@ -67,12 +67,8 @@ function startTest(evt) {
   evt.preventDefault();
   createBackend();
   if (!isPluginLoaded()) {
```

Comment:
Could perhaps reduce complexity through bringing the isPluginLoaded functions here?